HawiCaesar/smart-goals-api
# Ft app tests 149441167

### What does this PR do?
- Wrote more tests to cover the entire application.
- Add Code Climate Badges and Coveralls Badge
- Changed the develop branch travis badge to use develop branch

### How can I test this?
You can check out the badges [here](https://github.com/HawiCaesar/smart-goals-api/tree/ft-app-tests)
You can clone the ft-app-tests branch run the following in the the terminal(after making a virtual environment and a virtual environment wrapper)

`pip install -r requirements.txt`
`nosetests tests/`

The tests are ran automatically
